ADR/####-slsa-source-provenance-verification.md

```diff
@@ -0,0 +1,133 @@
+   - VSAs attest to source control practices (e.g., two-person review, branch protection, audit logs)
+
+2. **Source provenance verification task** (in Konflux pipeline):
+   - A Konflux pipeline task reads the source VSA from git notes for the cloned commit
+   - The task accepts a public key parameter (provided via IntegrationTestScenario configuration) and verifies the source VSA signature against that key
+   - The task verifies the VSA contents (format, subject matching the commit, etc.)
+   - The task pushes the source VSA to the OCI registry as a separate artifact
+   - The task exposes the pullspec of the OCI representation of the VSA (digest reference) in its task results using `*_ARTIFACT_OUTPUTS` format
+   - The task records the public key used for verification in its results
+
+3. **Build provenance chain** (via Tekton Chains):
+   - Tekton Chains observes the task results containing the source VSA artifact reference
+   - Chains generates and signs a SLSA build provenance attestation **about** the source provenance attestation
+   - This creates an attestation chain: `Build Provenance → Source VSA → Source Commit`
+
+4. **Verification in Conforma**:
+   - Conforma policies can verify source provenance by checking for the presence of the attestation chain
+   - Conforma trusts Tekton Chains' signatures (existing trust root)
+   - Conforma reads the public key from the test attestation (recorded by the verification task) and verifies it matches the expected signer identity for the source repository
+   - Through the chain, Conforma gains transitive trust in the source VSA's signature verification (performed by the task) without needing to re-verify the signature itself
+   - This bootstraps trust in source provenance through our existing build-provenance infrastructure
+
+### Rationale
+
+This approach provides several advantages:
+
+- **Preserves existing trust model**: Conforma continues to trust Tekton Chains as its primary attestation source
+- **Leverages ADR 48**: Using the movable test pattern makes source verification easier to re-run independently of builds
+- **Enables gradual adoption**: Teams can adopt source provenance incrementally without breaking existing workflows
+
+### Integration Points
+
+- **Verification task**: New task to read, verify, and republish source VSAs to OCI registry
+- **Tekton Chains**: Existing Chains infrastructure generates attestations about the verification task
+- **Enterprise Contract**: Conforma policies updated to verify the attestation chain
+- **Trusted Artifacts** ([ADR 36](0036-trusted-artifacts.md)): Source VSA artifacts follow the similar OCI storage patterns to other trusted artifacts
+
+## Consequences
+
+### Positive
+
+- **Consistent trust model**: All Conforma verification continues through Tekton Chains attestations
+- **Storage model alignment**: Source attestations stored as git notes (close to source), build attestations about them stored in OCI (close to artifacts)
```

Comment:
Should be removed for the same reasons as https://github.com/konflux-ci/architecture/pull/268#discussion_r2419811989 I think

Reply:
Done!